src/config.ts

```diff
@@ -0,0 +1,358 @@
+import moment from 'moment'
```

Comment:
Maybe name this file something else since we have config.js in the other folder? Also add a file comment

Reply:
I added the file comment.

I'm not sure I want to rename this one. I'd rather rename the config.js file. Maybe we name it chesster_config.js or something. Is it ok if I just file an issue about that, rather than fixing it right now?

Reply:
The primary reason is that I think I want to make even more improvements at the same time, so dealing with the rename in isolation is unnecessary work.